zomes/timed/coordinator/src/lib.rs

```diff
@@ -42,6 +42,31 @@ fn get_timed_entries_local() -> ExternResult<Vec<Record>> {
     Ok(records)
 }
 
+#[hdk_extern]
```

Comment:
New zome function required for zero arc nodes to be able to get entries from full arc nodes.